rust/datafusion/src/execution/aggregate.rs

```diff
@@ -142,9 +142,14 @@ impl AggregateFunction for MinFunction {
                 (Some(ScalarValue::Float64(a)), Some(ScalarValue::Float64(b))) => {
                     Some(ScalarValue::Float64(a.min(*b)))
                 }
-                _ => panic!("unsupported data type for MIN"),
+                _ => {
+                    return Err(ExecutionError::ExecutionError(
+                        "unsupported data type for MIN".to_string(),
```

Comment:
nit: You mix starting your error messages with `unsupported...` and 'Unsupported...`